scripts/build-snarkyjs-node.sh

```diff
@@ -9,7 +9,8 @@ export DUNE_USE_DEFAULT_LINKER="y"
 
```

Comment:
Can you add the typical prefix to the top of this so we catch all the errors:

```
#!/bin/bash

set -e
```

Then you can get rid of that `|| exit 1` and we'll catch more errors

Reply:
oh this is what this does? cool!